src/Uno.UI.Runtime.Skia/GLCanvasElement.GLVisual.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.UI.Composition;
+using SkiaSharp;
+using Uno.UI.Composition;
+
+namespace Microsoft.UI.Xaml.Controls;
+
+public abstract partial class GLCanvasElement
+{
+	private class GLVisual(GLCanvasElement owner, Compositor compositor) : Visual(compositor)
+	{
+		private readonly SKPixmap _pixmap = new SKPixmap(new SKImageInfo((int)owner._width, (int)owner._height, SKColorType.Bgra8888), owner._pixels);
```

Comment:
@ramezgerges This fixes the transparency issue:

```suggestion
		private readonly SKPixmap _pixmap = new SKPixmap(new SKImageInfo((int)owner._width, (int)owner._height, SKColorType.Bgra8888, SKAlphaType.Unpremul), owner._pixels);
```